browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -89,6 +99,88 @@ void BraveToolbarView::ShowBookmarkBubble(
     star_view->OnBubbleWidgetCreated(bubble_widget);
 }
 
+// Returns right-margin
+int BraveToolbarView::SetLocationBarBounds(const int available_width,
+                                          const int location_bar_min_width,
+                                          const int next_element_x,
+                                          const int element_padding) {
+  DCHECK(initialized_);
+  DCHECK(is_display_mode_normal());
+  // Allow the option of the LocationBar having horizontal margin.
+  // With this option, we support a dynamic percentage of margin, with
+  // a maximum width.
+  const bool restrict_location_bar_width = !location_bar_is_wide_.GetValue();
+  const int location_bar_max_width = restrict_location_bar_width
+                                      ? 1080
```

Comment:
Does DPI matter at all here? For instance, I run Windows at a 2x scale factor.

Reply:
Afaik, view framework uses dpi. So this wouldn't have a problem

Reply:
BTW, are there no UI issue when user uses big screen?